test/infrastructure/docker/config/crd/bases/infrastructure.cluster.x-k8s.io_dockermachinetemplates.yaml

```diff
@@ -99,6 +99,46 @@ spec:
         type: object
     served: true
     storage: false
```

Comment:
Looks like v1alpha3/v1alpha4 is now twice in the array. (Based on the other changes in this PR, I would have expected this file to stay the same)

Reply:
I ran into the same problem in https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/1389. The way I dealt with it was to rename `AzureMachineTemplate` to `AzureMachinePoolMachineTemplate` https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/1389/commits/afe9c203b223a842c0788a7eef5d8147f5dd0de2

Reply:
I made changes based on Cecile's suggestion, hopefully it should pass the tests now.